### PR TITLE
fix URLs, github actions

### PR DESCRIPTION
**Pull request by matbesancon**
Fix various small URLs to `jump-dev` and replace travis and appveyor with github actions

### CODE REVIEW

**Comment by matbesancon** [comment]
The tests seem to use an incorrect signature of MOI.optimize!
```
    # model = DiffOpt.Optimizer(...)

    sol = MOI.optimize!(model)

    x = sol.primal
    s = sol.slack
    y = sol.dual

```

in MOI_wrapper.jl.

Ping @AKS1996

**Comment by matbesancon** [comment]
The primal and dual access parts can be fixed quickly, but the slack requires some additional computation

**Comment by matbesancon** [comment]
Fixing CI does not make sense while the tests are all still breaking, so tackling the immediate issues
#46 #47

**Comment by matbesancon** [comment]
I am wondering if we should merge this one first and then fix the later issues

**Comment by matbesancon** [comment]
OK all tests are passing except for the ones linked to SCS not supporting a given constraint type, i.e. linked to
https://github.com/jump-dev/DiffOpt.jl/pull/37
I would suggest merging this and getting #37 merged soon. The other way would be to comment out all conic tests (not ideal) or merge #37 into this

**Review by blegat — APPROVED** [review]
Let's merge

**Comment by matbesancon** [comment]
I made the tests pass, I'll just update for GH actions to add MatrixOptI

**Comment by matbesancon** [comment]
Not sure about the GH syntax, we'll see

**Comment by matbesancon** [comment]
So good news is some tests pass and the actions are finally set up.
Bad news is there are still some failing tests

**Comment by matbesancon** [comment]
I suspect the test problems not to be strongly convex -> multiple possible solutions found by different solvers?

**Comment by matbesancon** [comment]
There seems to be a windows-only escaping issue:
https://github.com/jump-dev/DiffOpt.jl/pull/45/checks?check_run_id=1517097034
when Pkg.dev-ing MatOI. @odow have you encountered something similar?

**Comment by matbesancon** [comment]
I get rather large numerical errors on a slack test:
```
s - expected_s:
 1.1734166677389504e-15
 0.13905758625384124
 4.715214951276747e-16
 0.1775338024275932
 6.69285964927494e-17
 5.279372035384908e-16
 3.843926470251321e-17
 0.2625892873538369
 0.04306063761061951
 0.1012750010480592
 0.0
```

**Comment by matbesancon** [comment]
I think we're close to something working for docs too, once this is the case this should be merged, I still don't know how to make windows work with URL properly without breaking on Linux. And the 32-bits issue is related to an upstream package

**Comment by matbesancon** [comment]
alright good to merge

**Comment by matbesancon** [comment]
Ok windows passing too

**Comment by matbesancon** [comment]
replaces #37

**Comment by matbesancon** [comment]
bump

File: src/utils.jl [hunk 1]
```diff
@@ -88,54 +86,78 @@ function get_problem_data(model::MOI.AbstractOptimizer)
...
+
+        q = MOI.coefficient.(objective_function.affine_terms)
+    end
+
+    return Q, q, G, h, A, b, nz, var_list, nineq, ineq_con_idx, neq, eq_con_idx
+end
+
+# might slow down computation
+# need to find a faster way
+function CSRToCSC(B::MatOI.SparseMatrixCSRtoCSC{Int64})
```
Comment (blegat) [inline src/utils.jl:154]: In fact, it's easier to get the conic form for CSC A, we do it in CSR in MatOI as this is what SCS needs but we can add a mutable CSC matrix in MatOI too quite easily so that DiffOpt can use that but let's leave that for another PR

**Review by blegat — APPROVED** [review]

File: src/utils.jl (hunk 1 above)
Comment (matbesancon) [reply src/utils.jl:154]: maybe document it in an issue?

**Comment by matbesancon** [comment]
See https://github.com/oxfordcontrol/OSQP.jl/issues/68 for removing 32 bits for now
